pkgs/applications/science/programming/scyther/default.nix

```diff
@@ -0,0 +1,84 @@
+{ stdenv, lib, fetchFromGitHub, glibc, flex, bison, python27Packages, graphviz, cmake
+, includeGUI ? false
+, includeProtocols ? true
+}:
+let
+  version = "v1.1.3";
```

Comment:
This version string is invalid. See `builtins.parseDrvName`. A version string must begin with a digit.